backend/utils_segment_whole_slide.py

```diff
@@ -35,7 +44,7 @@ def get_saved_contours(level_array):
 def prepare_tile_info(
     x0: int, y0: int,
     w0: int, h0: int,
-    tile_size: int,
+    tile_size:  TILE_SIZE,
```

Comment:
Incorrect: `TILE_SIZE` is not a type!
```suggestion
    tile_size:  int = TILE_SIZE,
```

Reply:
Changed back to int.